### DISCUSSION TITLE
Resizing page - content placement

### DISCUSSION THREAD

**Discussion by jjennings955** [pymupdf/PyMuPDF#2282] [Looking for help] [answered]
Hello,

I'm resizing pages by updating the mediabox to have 200 extra pixels of height. This puts the extra height at the top of the page. How do I shift the contents of the page up, so I can have the extra space at the bottom?

So right now I have something like this
```    doc = fitz.open(args.input)
    for i, page in enumerate(doc):
        # resize the page to add a comment section at the bottom
        rect = fitz.Rect(0, 0, page.rect.width, page.rect.height + 200)
        page.set_mediabox(rect)
```

Thanks in advance!

**Comment by JorjMcKie** [comment, marked as answer]
You can set the MediaBox with negative coordinates, too! For example:
```python
doc=fitz.open()
page=doc.new_page()
cb=page.cropbox
page.draw_rect(page.rect,color=(1,0,0))
Point(0.0, 0.0)
page.set_mediabox(cb+(0,-200,0,0))
print(doc.xref_object(4))
<<
  /Type /Page
  /MediaBox [ 0 -200 595 842 ]   # this is legal!
  /Rotate 0
  /Resources 3 0 R
  /Parent 2 0 R
  /Contents [ 5 0 R ]
>>
doc.save("x.pdf")
page.cropbox
Rect(0.0, 0.0, 595.0, 1042.0)
```
looks like this:
![grafik](https://user-images.githubusercontent.com/8290722/224473668-1c988c41-64a6-44f3-878b-4daee4e86731.png)

**Reply by jjennings955** [reply]
Thanks!
